terraform/aws/modules/uc_metastore/README.md

```diff
@@ -0,0 +1,37 @@
+<!-- BEGIN_TF_DOCS -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_databricks"></a> [databricks](#requirement\_databricks) | ~>0.3.12 |
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_databricks"></a> [databricks](#provider\_databricks) | ~>0.3.12 |
+
+## Modules
+
+No modules.
+
+## Resources
+
+| Name | Type |
+|------|------|
+| [databricks_metastore.unity](https://registry.terraform.io/providers/databrickslabs/databricks/latest/docs/resources/metastore) | resource |
```

Comment:
docs are not published yet (they will be on public preview).

Reply:
joy of auto-generated doc - I've removed them for now